### PR TITLE
Issue #397 solved

### PR DESCRIPTION
**Pull request by sarthak-g**
Screenshot from 2019-11-17 08-52-35

changed clear icon in game of life activity as mentioned in issue #397

### CODE REVIEW

File: activities/Calculate.activity/js/calculate-activity.js [hunk 1]
```diff
@@ -203,6 +203,7 @@ function calcEqualClick() {
     launchCalculation(calcInputValue, labelValue);
   }
   CalculateApp.data.calculationDone = true
+  CalculateApp.focus();
```
Comment (llaske) [inline activities/Calculate.activity/js/calculate-activity.js]: This fix is not related to this issue

**Comment by llaske** [comment]
Nice. But I guess that your development environment is not up to date.

**Comment by sarthak-g** [comment]
I've updated my environment and raised pull request #448
